features/nanostack/coap-service/test/coap-service/unittest/coap_connection_handler/test_coap_connection_handler.c

```diff
@@ -394,6 +394,10 @@ bool test_socket_api_callbacks()
     memset(&buf, 1, 16);
 
     socket_callback_t *sckt_data = (socket_callback_t *)malloc(sizeof(socket_callback_t));
+    if(NULL == sckt_data)
```

Comment:
please fix the formatting, consistency with the coding style in the file.

Reply:
like this?
```
if(!sckt_data)
{
        return false;
}
```

Reply:
if you look at the file, it follows Mbed OS coding style. In this case

```
if (test) {
    do_something;
}
```

Reply:
Changed as with the coding style in the file, thanks you.